web/src/components/landing-header.js

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import {Container,
+        Grid,
+        Typography
+} from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import LandiingViewButton from './landing-view-button';
+
+const landing_bg = require('../assets/landing-bg.jpg');
+const benefit = [
+  require('../assets/benefit1.png'),
+  require('../assets/benefit2.png'),
+  require('../assets/benefit3.png')
+];
+
+// Main Header css style
+const mainHeaderStyles = makeStyles(theme => ({
+  root: {
+    width: '100%',
+    backgroundImage: `url(${landing_bg})`,
+    backgroundSize: 'cover',
+    color:'white',
+    textAlign: 'center',
+    height: 400,
+    justifyContent: 'center',
+    alignItems: 'center'
+  },
+
+}));
+
+// Sub Explanation css style
```

Comment:
What is Sub Explanation?

Reply:
It's sub explanation(== product benefit?) under main header. Is that name not good?